ui/webui/resources/polymer_overriding.ts

```diff
@@ -334,7 +334,11 @@ function BraveDefineCustomElements(name, component, options, useIgnoreList = tru
       delete allBehaviorsMap[name]
     }
   }
-  oldDefine.call(this, name, component, options)
+
+  // Allow a tick for overrides to be registered before we define the component.
+  setTimeout(() => {
```

Comment:
I'd run into this issue before with other components but finally got to the root of it here. This was triggering a race between registering the style overrides and defining the component - I think I triggered it here by importing the file being overridden.

Reply:
This will increase the number of places we can use RegisterStyleOverride from